src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from "react";
+import "./styles.css";
+import axios from "axios";
+import { Snackbar } from "./Components/Snackbar";
+import { useData } from "./Contexts/dataProvider";
+import { NavBar } from "./Components/NavBar";
+import { url } from "./util";
+import { Router } from "./Components/Router";
+
+export default function App() {
+  const {
+    state: { snackbarText },
+    dispatch
+  } = useData();
+
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axios.get(url);
+        localStorage.setItem("data", JSON.stringify(response.data.products));
+        const data = JSON.parse(localStorage?.getItem("data"));
```

Comment:
Hi Kajol,
Could you please explain this line?
Here we are saving the products in the localStorage. But I was wondering where are we exactly using that data?

Reply:
I am just fetching it to products list. So if you refresh the page then data will appear on the page quickly, that's it.
So do you suggest that it is unnecessary and might effect performance, so I should remove it?